ebodac/src/test/java/org/motechproject/ebodac/uitest/page/BookingAppScreeningPage.java

```diff
@@ -150,14 +150,13 @@ public boolean bookingIdExistsOnPage(String id) {
         }
     }
 
-    public boolean isFirstBookingOK(ArrayList<Date> dates) {
+    public boolean isFirstBookingOK(ArrayList<LocalDate> dates) {
         try {
             WebElement firstVisitElement = findElement(FIRST_VISIT);
             if (firstVisitElement == null) {
                 return true;
             } else {
-                long dateparse = Date.parse((firstVisitElement.getAttribute("title")).toString());
-                Date date = new Date(dateparse);
+                LocalDate date = LocalDate.parse((firstVisitElement.getAttribute("title")).toString());
```

Comment:
It is better to use LocalDate.parse("date", DateTimeFormat.forPattern("pattern")), then you will be sure the date was parsed correctly, pattern can be "yyyy-MM-dd", you can see the patterns here http://joda-time.sourceforge.net/apidocs/org/joda/time/format/DateTimeFormat.html